integration_tests/grpc_rest/model/helpers.go

```diff
@@ -519,3 +556,65 @@ func GetModelForInvalidVidPid(suite *utils.TestSuite) {
 	// require.True(suite.T, sdkerrors.ErrInvalidRequest.Is(err))
 	suite.AssertNotFound(err)
 }
+
+func AddModelInHexFormat(suite *utils.TestSuite) {
+	// Alice and Bob are predefined Trustees
+	aliceName := testconstants.AliceAccount
+	aliceKeyInfo, err := suite.Kr.Key(aliceName)
+	require.NoError(suite.T, err)
+	aliceAccount, err := test_dclauth.GetAccount(suite, aliceKeyInfo.GetAddress())
+	require.NoError(suite.T, err)
+
+	bobName := testconstants.BobAccount
+	bobKeyInfo, err := suite.Kr.Key(bobName)
+	require.NoError(suite.T, err)
+	bobAccount, err := test_dclauth.GetAccount(suite, bobKeyInfo.GetAddress())
+	require.NoError(suite.T, err)
+
+	// Register new Vendor account
+	vendorName := utils.RandString()
+	var vid int32 = 0xA13
+	vendorAccount := test_dclauth.CreateAccount(
+		suite,
+		vendorName,
+		dclauthtypes.AccountRoles{dclauthtypes.Vendor},
+		vid,
+		aliceName,
+		aliceAccount,
+		bobName,
+		bobAccount,
+		testconstants.Info,
+	)
+	require.NotNil(suite.T, vendorAccount)
+
+	// Get all models
+	inputModels, err := GetModels(suite)
+	require.NoError(suite.T, err)
+
+	// New vendor adds first model
+	createFirstModelMsg := NewMsgCreateModel(0xA13, 0xA11, vendorAccount.Address)
```

Comment:
Avoid using magic constants!
It is better to assign them to variables, since it does not affect value for `int32` type:
```
var vid int32 = 0xA13
var pid int32 = 0xA11
```

Reply:
Done! ✅